packages/qwik/src/optimizer/src/plugins/vite.ts

```diff
@@ -558,23 +558,26 @@ export function qwikVite(qwikViteOpts: QwikVitePluginOptions = {}): any {
             fileName: Q_MANIFEST_FILENAME,
             source: clientManifestStr,
           });
+          const assetsDir = qwikPlugin.getOptions().assetsDir || '';
+          const useAssetsDir = !!assetsDir && assetsDir !== '_astro';
           const sys = qwikPlugin.getSys();
-          const filePath = sys.path.dirname(_.chunkFileNames as string);
           this.emitFile({
             type: 'asset',
-            fileName: sys.path.join(filePath, `q-bundle-graph-${manifest.manifestHash}.json`),
+            fileName: sys.path.join(
+              useAssetsDir ? assetsDir : '',
```

Comment:
Looks like we could combine the `useAssetsDir ? ...` logic into one parameter for both places 

Maybe something like `buildPath` or something alike

Reply:
I like it